src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -1312,11 +1311,11 @@ public static void MakeSureNoContainsChecksGoOutOfRange_StringComparison()
 
         [Fact]
         public static void ContainsUnknownComparisonType_StringComparison()
-        {                        
-            ReadOnlySpan<char> span = "456".AsSpan();
-            SpanTestHelpers.AssertThrows<ArgumentException, char>(span, (_span) => _span.Contains(_span, StringComparison.CurrentCulture - 1));
-            SpanTestHelpers.AssertThrows<ArgumentException, char>(span, (_span) => _span.Contains(_span, StringComparison.OrdinalIgnoreCase + 1));
-            SpanTestHelpers.AssertThrows<ArgumentException, char>(span, (_span) => _span.Contains(_span, (StringComparison)6));
+        {
+            string value = "456";
+            Assert.Throws<ArgumentException>(() => value.AsSpan().CompareTo(value.AsSpan(), StringComparison.CurrentCulture - 1));
+            Assert.Throws<ArgumentException>(() => value.AsSpan().CompareTo(value.AsSpan(), StringComparison.OrdinalIgnoreCase + 1));
+            Assert.Throws<ArgumentException>(() => value.AsSpan().CompareTo(value.AsSpan(), (StringComparison)6));
```

Comment:
Btw, @stephentoub, regarding your feedback around using `_span` as the name of a local (https://github.com/dotnet/corefx/pull/38865#discussion_r297207636), a bunch of System.Memory tests end up using `_span`. Should we do a similar cleanup there (outside this PR)?

E.g.:
https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Memory/tests/Binary/BinaryReaderUnitTests.cs#L162-L178
https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Memory/tests/MemoryMarshal/CastReadOnlySpan.cs#L40-L41
https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Memory/tests/Span/CopyTo.cs#L95
https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Memory/tests/TestHelpers.cs#L33

Reply:
I would prepare PR for this